seedbuilder/areas.ori

```diff
@@ -1665,53 +1673,68 @@ home: BashTree 521 326
 	conn: BashTreeDoorClosed
 		casual Open
 	conn: UpperGinsoRedirectArea
-		casual Bash
-		casual ChargeJump Dash DoubleJump
+		casual Bash WallJump
+		casual Bash Climb
+		casual Bash DoubleJump
```

Comment:
Can't get up to the redirect puzzle after the fronkey, needs DJump WI or Lure the fronkey which is not casual

Reply:
Confused on what this means. Is this not fixed? Every casual path here has either height or wall interaction skill.

Reply:
![2023-11-15_10-28-47](https://github.com/sparkle-preference/ori_rando_server/assets/32931178/c1560ad4-91ee-4f33-bf1f-8b2bc4915cb2)
`casual Bash DoubleJump`
This is the section I'm concerned about. If you bash the fronkey to get here, it stays alive and doesn't respawn. That means that you don't have anything to bash unless you lure the fronkey into the spikes to kill it and make it respawn next to you.

Reply:
Ah I understand now. Hmm, the nice thing about this lure is that you don't have to lure the fronkey from off the floor (fall off right, double jump back up left to get his attention). Once you get near the edge of the floor, it gets attracted. One factor that helps potentially keep this small lure in casual. I'll see what others say about this.

Reply:
I don't feel like using the fronkey for both parts of that path should be in casual. And it's not like we gain anything out of it in practice anyway.

Reply:
Sounds good, I will remove it.